wagtail/wagtailadmin/static_src/wagtailadmin/scss/components/_streamfield.scss

```diff
@@ -147,56 +171,59 @@ li.sequence-member {
 // Controls for ordering and deletion of items in a sequence
 .sequence-controls {
     @include transition(opacity 0.2s ease);
-    border-radius: 2px 0 0;
-    border: 1px solid darken($color-input-focus, 10%);
-    border-width: 1px 1px 0;
-    opacity: 0;
-    visibility: hidden;
-    position: absolute;
-    top: -30px;
+    height: 30px;
+    overflow: visible;
     right: 0;
+    width: 100%;
     z-index: 1;
-    overflow: visible;
-    background-color: $color-input-focus;
-    padding: 0 0 0 1em;
-    height: 30px;
 
     h3 {
-        margin: 0;
-        font-size: 0.7rem;
         display: inline-block;
-        margin-right: 1em;
+        font-size: 0.8rem;
         line-height: 1.8rem;
+        padding: 0 20px;
 
         label {
```

Comment:
As far as I can see this label is only used for its ARIA value – would you have the courage to refactor this to remove this element (styling the `h3` instead, ideally adding a class to it), keeping the ARIA labelling with a `aria-describedby` attribute on the heading?